docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       context: .
       dockerfile: Dockerfile-dev
     ports:
-      - "80:80"
+      - "8000:80"
```

Comment:
I think this was added by mistake

Reply:
I sure was, have changed it back now. ↩️ 🎉